src/psyclone/tests/psyad/adjoint_visitor_test.py

```diff
@@ -556,7 +607,6 @@ def test_loop_node_active(fortran_reader, fortran_writer, in_bounds,
     '''
     code = TL_LOOP_CODE.replace("lo,hi,step", in_bounds)
     tl_psyir = fortran_reader.psyir_from_source(code)
-    tl_loop = tl_psyir.walk(Loop)[0]
     adj_visitor = AdjointVisitor(["a", "b", "c"])
```

Comment:
Since we're here, please could you change line 615 to use an f-string?

Reply:
Modified.